src/PIL/ImageCms.py

```diff
@@ -247,7 +246,7 @@ def __init__(self, profile: str | SupportsRead[bytes] | core.CmsProfile) -> None
                         self.profile = core.profile_frombytes(f.read())
                     return
             self.filename = profile
-            self.profile = core.profile_open(profile)
+            self.profile: core.CmsProfile = core.profile_open(profile)
```

Comment:
```suggestion
            self.profile = core.profile_open(profile)
```
Testing with PyCharm, I don't see any advantage to explicitly type hinting this variable.